solr/solr-ref-guide/modules/query-guide/pages/text-to-vector.adoc

```diff
@@ -382,7 +382,7 @@ You still define two chains, but this time the 'vectorisation' one only includes
 
 [NOTE]
 ====
-Partial updates are processed by the `DistributedUpdateProcessorFactory`. For this reason, it must be placed at the beginning of the UpdateRequestProcessorChain, before the `TextToVectorUpdateProcessorFactory`.
+Since Partial updates are processed by the `DistributedUpdateProcessorFactory`, this processor must be placed at the beginning of the UpdateRequestProcessorChain, before the `TextToVectorUpdateProcessorFactory`.
```

Comment:
It's still confusing IMO -- "this" is ambiguous.  I suggest the following:

> Since partial updates are resolved by `DistributedUpdateProcessorFactory`, be sure to place `TextToVectorUpdateProcessorFactory` afterwards so that it sees normal/complete documents.